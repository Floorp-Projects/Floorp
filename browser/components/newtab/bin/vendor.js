#!/usr/bin/env node
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */

/* eslint-disable no-console */

const { cp, set } = require("shelljs");
const path = require("path");

const filesToVendor = {
  // XXX currently these two licenses are identical.  Perhaps we should check
  // in case that changes at some point in the future.
  "react/LICENSE": "REACT_AND_REACT_DOM_LICENSE",
  "react/umd/react.production.min.js": "react.js",
  "react/umd/react.development.js": "react-dev.js",
  "react-dom/umd/react-dom.production.min.js": "react-dom.js",
  "react-dom/umd/react-dom.development.js": "react-dom-dev.js",
  "react-dom/umd/react-dom-server.browser.production.min.js":
    "react-dom-server.js",
  "react-redux/LICENSE.md": "REACT_REDUX_LICENSE",
  "react-redux/dist/react-redux.min.js": "react-redux.js",
  "react-transition-group/dist/react-transition-group.min.js":
    "react-transition-group.js",
  "react-transition-group/LICENSE": "REACT_TRANSITION_GROUP_LICENSE",
};

set("-v"); // Echo all the copy commands so the user can see what's going on
for (let srcPath of Object.keys(filesToVendor)) {
  cp(
    path.join("node_modules", srcPath),
    path.join("vendor", filesToVendor[srcPath])
  );
}

console.log(`
Check to see if any license files have changed, and, if so, be sure to update
https://searchfox.org/mozilla-central/source/toolkit/content/license.html`);
