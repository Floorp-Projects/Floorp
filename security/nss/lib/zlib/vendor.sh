#!/usr/bin/env bash
# Vendor a new version of zlib.
#
# Note: This script doesn't remove files if they are removed in the zlib release.

set -e

if [[ $# -lt 1 ]]; then
    echo "Usage: $0 <version>" 1>&2
    exit 2
fi

version="$1"
ztmp="zlib.$version"

cd "$(dirname "$0")"
../../fuzz/config/git-copy.sh https://github.com/madler/zlib "v$version" "$ztmp"
fullversion="$version ("$(cat "$ztmp"/.git-copy)")"
sed -i -e 's/^Version: .*/Version: '"$fullversion"'/' README.nss

prune=(
    .git-copy
    .gitignore
    CMakeLists.txt
    ChangeLog
    FAQ
    INDEX
    Makefile
    Makefile.in
    amiga
    configure
    contrib
    doc
    example.c
    examples
    make_vms.com
    minigzip.c
    msdos
    nintendods
    old
    os400
    qnx
    test
    treebuild.xml
    watcom
    win32
    zconf.h.cmakein
    zconf.h.in
    zlib.3
    zlib.3.pdf
    zlib.map
    zlib.pc.cmakein
    zlib.pc.in
    zlib2ansi
)
for i in "${prune[@]}"; do rm -rf "$ztmp"/"$i"; done
for i in "$ztmp"/*; do mv "$i" .; done
rmdir "$ztmp"
