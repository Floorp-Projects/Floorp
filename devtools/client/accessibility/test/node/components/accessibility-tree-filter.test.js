/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
"use strict";

const { mount } = require("enzyme");

const { createFactory } = require("devtools/client/shared/vendor/react");
const Provider = createFactory(
  require("devtools/client/shared/vendor/react-redux").Provider
);

const MenuButton = require("devtools/client/shared/components/menu/MenuButton");
const ConnectedAccessibilityTreeFilterClass = require("devtools/client/accessibility/components/AccessibilityTreeFilter");
const AccessibilityTreeFilterClass =
  ConnectedAccessibilityTreeFilterClass.WrappedComponent;
const AccessibilityTreeFilter = createFactory(
  ConnectedAccessibilityTreeFilterClass
);
const {
  checkMenuItem,
  setupStore,
} = require("devtools/client/accessibility/test/node/helpers");

const {
  AUDIT,
  AUDITING,
  FILTERS,
  FILTER_TOGGLE,
} = require("devtools/client/accessibility/constants");

function checkToggleFilterCheckbox(wrapper, filter) {
  const filterInstance = wrapper.find(AccessibilityTreeFilterClass).instance();
  filterInstance.toggleFilter = jest.fn();
  filter.click();
  expect(filterInstance.toggleFilter.mock.calls.length).toBe(1);
}

function getMenuItems(wrapper, selector) {
  const menuButton = wrapper.find(MenuButton);
  // Focusing on the menu button will trigger rendering of the HTMLTooltip with
  // the menu list.
  menuButton
    .childAt(0)
    .getDOMNode()
    .focus();

  return menuButton
    .instance()
    .tooltip.panel.querySelectorAll(`.menuitem ${selector}`);
}

function checkFiltersState(wrapper, expected) {
  const filters = getMenuItems(wrapper, ".filter");
  for (let i = 0; i < filters.length; i++) {
    checkMenuItem(filters[i], {
      checked: expected.filters[i].active,
      label: expected.filters[i].text,
      disabled: expected.filters[i].disabled,
    });
  }
}

describe("AccessibilityTreeFilter component:", () => {
  it("audit filter not filtered", () => {
    const store = setupStore();
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    const accTreeFilter = wrapper.find(AccessibilityTreeFilterClass);
    const toolbar = accTreeFilter.childAt(0);

    expect(wrapper.html()).toMatchSnapshot();
    expect(accTreeFilter.children().length).toBe(1);
    expect(toolbar.is("div")).toBe(true);
    expect(toolbar.prop("role")).toBe("group");

    checkFiltersState(wrapper, {
      filters: [
        { active: true, disabled: false, text: "None" },
        { active: false, disabled: false, text: "All Issues" },
        {
          active: false,
          disabled: false,
          text: "Contrast",
        },
        {
          active: false,
          disabled: false,
          text: "Keyboard",
        },
        {
          active: false,
          disabled: false,
          text: "Text Labels",
        },
      ],
    });
  });

  it("audit filters filtered", () => {
    const store = setupStore({
      preloadedState: {
        audit: {
          filters: {
            [FILTERS.ALL]: true,
            [FILTERS.CONTRAST]: true,
            [FILTERS.KEYBOARD]: true,
            [FILTERS.TEXT_LABEL]: true,
          },
          auditing: [],
        },
      },
    });
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    expect(wrapper.html()).toMatchSnapshot();
    checkFiltersState(wrapper, {
      filters: [
        { active: false, disabled: false },
        { active: true, disabled: false },
        { active: true, disabled: false },
        { active: true, disabled: false },
        { active: true, disabled: false },
      ],
    });
  });

  it("audit all filter not filtered auditing", () => {
    const store = setupStore({
      preloadedState: {
        audit: {
          filters: {
            [FILTERS.ALL]: false,
          },
          auditing: [FILTERS.ALL],
        },
      },
    });
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    expect(wrapper.html()).toMatchSnapshot();
    checkFiltersState(wrapper, {
      filters: [
        { active: true, disabled: false, text: "None" },
        { active: false, disabled: true, text: "All Issues" },
      ],
    });
  });

  it("audit other filter not filtered auditing", () => {
    const store = setupStore({
      preloadedState: {
        audit: {
          filters: {
            [FILTERS.ALL]: false,
            [FILTERS.CONTRAST]: false,
            [FILTERS.KEYBOARD]: false,
            [FILTERS.TEXT_LABEL]: false,
          },
          auditing: [FILTERS.CONTRAST],
        },
      },
    });
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    expect(wrapper.html()).toMatchSnapshot();
    checkFiltersState(wrapper, {
      filters: [
        { active: true, disabled: true },
        { active: false, disabled: false },
        { active: false, disabled: true },
        { active: false, disabled: false },
        { active: false, disabled: false },
      ],
    });
  });

  it("audit all filter filtered auditing", () => {
    const store = setupStore({
      preloadedState: {
        audit: {
          filters: {
            [FILTERS.ALL]: true,
          },
          auditing: [FILTERS.ALL],
        },
      },
    });
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    const filters = getMenuItems(wrapper, ".filter");
    expect(wrapper.html()).toMatchSnapshot();
    checkMenuItem(filters[1], { checked: true, disabled: true });
  });

  it("audit other filter filtered auditing", () => {
    const store = setupStore({
      preloadedState: {
        audit: {
          filters: {
            [FILTERS.ALL]: false,
            [FILTERS.CONTRAST]: true,
            [FILTERS.KEYBOARD]: false,
            [FILTERS.TEXT_LABEL]: false,
          },
          auditing: [FILTERS.CONTRAST],
        },
      },
    });
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    expect(wrapper.html()).toMatchSnapshot();
    checkFiltersState(wrapper, {
      filters: [
        { active: false, disabled: true },
        { active: false, disabled: false },
        { active: true, disabled: true },
        { active: false, disabled: false },
        { active: false, disabled: false },
      ],
    });
  });

  it("toggle filter", () => {
    const store = setupStore();
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    const filters = getMenuItems(wrapper, ".filter");

    expect(wrapper.html()).toMatchSnapshot();
    for (const filter of filters) {
      checkToggleFilterCheckbox(wrapper, filter);
    }
  });

  it("render filters after state changes", () => {
    const store = setupStore();
    const wrapper = mount(
      Provider({ store }, AccessibilityTreeFilter({ toolboxDoc: document }))
    );
    const tests = [
      {
        expected: {
          filters: [
            { active: true, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
          ],
        },
      },
      {
        action: {
          type: AUDITING,
          auditing: Object.values(FILTERS),
        },
        expected: {
          filters: [
            { active: true, disabled: true },
            { active: false, disabled: true },
            { active: false, disabled: true },
            { active: false, disabled: true },
            { active: false, disabled: true },
          ],
        },
      },
      {
        action: {
          type: AUDIT,
          response: [],
        },
        expected: {
          filters: [
            { active: true, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
          ],
        },
      },
      {
        action: {
          type: FILTER_TOGGLE,
          filter: FILTERS.ALL,
        },
        expected: {
          filters: [
            { active: false, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
      {
        action: {
          type: FILTER_TOGGLE,
          filter: FILTERS.CONTRAST,
        },
        expected: {
          filters: [
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
      {
        action: {
          type: AUDITING,
          auditing: [FILTERS.CONTRAST],
        },
        expected: {
          filters: [
            { active: false, disabled: true },
            { active: false, disabled: false },
            { active: false, disabled: true },
            { active: true, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
      {
        action: {
          type: AUDIT,
          response: [],
        },
        expected: {
          filters: [
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
      {
        action: {
          type: FILTER_TOGGLE,
          filter: FILTERS.CONTRAST,
        },
        expected: {
          filters: [
            { active: false, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
      {
        action: {
          type: FILTER_TOGGLE,
          filter: FILTERS.NONE,
        },
        expected: {
          filters: [
            { active: true, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
          ],
        },
      },
      {
        action: {
          type: AUDITING,
          auditing: [FILTERS.TEXT_LABEL],
        },
        expected: {
          filters: [
            { active: true, disabled: true },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: true },
          ],
        },
      },
      {
        action: {
          type: AUDIT,
          response: [],
        },
        expected: {
          filters: [
            { active: true, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
          ],
        },
      },
      {
        action: {
          type: FILTER_TOGGLE,
          filter: FILTERS.TEXT_LABEL,
        },
        expected: {
          filters: [
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: false, disabled: false },
            { active: true, disabled: false },
          ],
        },
      },
    ];

    for (const test of tests) {
      const { action, expected } = test;
      if (action) {
        store.dispatch(action);
        wrapper.update();
      }

      expect(wrapper.html()).toMatchSnapshot();
      checkFiltersState(wrapper, expected);
    }
  });
});
