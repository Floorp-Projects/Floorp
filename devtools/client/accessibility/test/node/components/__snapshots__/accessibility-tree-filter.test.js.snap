// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessibilityTreeFilter component: audit all filter filtered auditing 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">All Issues</button></div>"`;

exports[`AccessibilityTreeFilter component: audit all filter not filtered auditing 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: audit filter not filtered 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: audit filters filtered 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">All Issues</button></div>"`;

exports[`AccessibilityTreeFilter component: audit other filter filtered auditing 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">Contrast</button></div>"`;

exports[`AccessibilityTreeFilter component: audit other filter not filtered auditing 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 2`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 3`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 4`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">All Issues</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 5`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">Keyboard, Text Labels</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 6`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">Keyboard, Text Labels</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 7`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">Keyboard, Text Labels</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 8`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">All Issues</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 9`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 10`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 11`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;

exports[`AccessibilityTreeFilter component: render filters after state changes 12`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">Text Labels</button></div>"`;

exports[`AccessibilityTreeFilter component: toggle filter 1`] = `"<div role=\\"group\\" class=\\"accessibility-tree-filters\\" aria-labelledby=\\"accessibility-tree-filters-label\\"><span id=\\"accessibility-tree-filters-label\\" role=\\"presentation\\">Check for issues:</span><button class=\\"devtools-button badge toolbar-menu-button filters\\" aria-expanded=\\"false\\" aria-haspopup=\\"menu\\" aria-controls=\\"accessibility-tree-filters-menu\\">None</button></div>"`;
